lib/core/src/lib/viewer/components/viewer.component.spec.ts

```diff
@@ -115,6 +115,23 @@ describe('ViewerComponent', () => {
 
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set alt attribute to originalMimeType when originalMimeType is provided', () => {
+            component.originalMimeType = 'image/png';
+            fixture.detectChanges();
+            const altAttribute = fixture.nativeElement.querySelector('.adf-viewer__mimeicon').getAttribute('alt');
+            expect(altAttribute).toBe('image/png');
+        });
+
+        it('should set src attribute based on originalMimeType when originalMimeType is provided', () => {
```

Comment:
Could you please also add opposite test where originalMimeType is not provided and mimeType is used instead?

Reply:
The opposite test case has been added.